internal/extsvc/jvmpackages/coursier/coursier.go

```diff
@@ -62,29 +59,29 @@ func FetchByteCode(ctx context.Context, config *schema.JVMPackagesConnection, de
 	return runCoursierCommand(
 		ctx,
 		config,
-		"fetch", "--intransitive",
-		dependency.CoursierSyntax(),
+		// NOTE: make sure to update the method `coursierScript` in
+		// vcs_syncer_jvm_packages_test.go if you change the arguments
+		// here. The test case assumes that the "--classifier sources"
+		// arguments appears at a specific index.
+		"fetch",
+		"--quiet", "--quiet",
+		"--intransitive", dependency.CoursierSyntax(),
 	)
 }
 
-func Exists(ctx context.Context, config *schema.JVMPackagesConnection, dependency reposource.MavenDependency) (bool, error) {
+func Exists(ctx context.Context, config *schema.JVMPackagesConnection, dependency reposource.MavenDependency) bool {
 	if dependency.IsJdk() {
-		lines, err := runCoursierCommand(
-			ctx,
-			config,
-			"java-home",
-			"--jvm",
-			dependency.Version,
-		)
-		return len(lines) > 0, err
+		sources, err := FetchSources(ctx, config, dependency)
+		return err == nil && len(sources) == 1
 	}
-	versions, err := runCoursierCommand(
+	_, err := runCoursierCommand(
 		ctx,
 		config,
-		"complete",
+		"resolve",
+		"--quiet", "--quiet",
 		dependency.CoursierSyntax(),
 	)
-	return len(versions) > 0, err
+	return err != nil
```

Comment:
```suggestion
	return err == nil
```

This caught me out big time while working on https://github.com/sourcegraph/sourcegraph/pull/22984 :^)

Reply:
Woah, nice catch. My bad 🙈

Reply:
`err != nil` is already built into muscle memory it's weird to write `err == nil`